components/calendar/demo/notice-calendar.md

```diff
@@ -1,10 +1,18 @@
 ---
 order: 2
-title: 通知事项日历演示
+title: 
+  zh-CN: 通知事项日历演示
+  en-US: A demo of notice-calendar
```

Comment:
`notice-calendar` not a word.

`A demo of notice-calendar` => `Notice Calendar`